build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
     `maven-publish`
 }
 
-version = "1.5.0+1.19"
+version = "1.5.1+1.19"
```

Comment:
Nit: this version bump should be to `1.6.0` because its a new 'feature' rather than just a bugfix